src/templates/components/rateYourExperience/index.test.tsx

```diff
@@ -1,65 +1,22 @@
 import { render } from '@testing-library/react'
 import { RateYourExperience } from './index'
-import userEvent from '@testing-library/user-event'
-import { waitFor } from '@testing-library/react'
-import { prettyDOM } from '@testing-library/react'
-import { screen } from '@testing-library/dom'
 
 describe('<RateYourExperience>', () => {
   test('renders <RateYourExperience />', () => {
-    const { queryByText } = render(<RateYourExperience />)
-
```

Comment:
Note that web components are notoriously impregnable in unit tests, so refactoring to use those has limited our unit testing capability.